airbyte-cdk/java/airbyte-cdk/typing-deduping/src/main/kotlin/io/airbyte/integrations/base/destination/typing_deduping/CatalogParser.kt

```diff
@@ -125,9 +133,10 @@ constructor(
     @VisibleForTesting
     fun toStreamConfig(stream: ConfiguredAirbyteStream): StreamConfig {
         if (stream.generationId == null) {
-            stream.generationId = 0
-            stream.minimumGenerationId = 0
-            stream.syncId = 0
+            // TODO set platform version
+            throw ConfigErrorException(
+                "You must upgrade your platform version to use this connector version. Either downgrade your connector or upgrade platform to X.Y.Z"
```

Comment:
NIT: put an actual version?

Reply:
platform is still working out a few final bugs, so we don't have a version number yet https://airbytehq-team.slack.com/archives/C06HBA0N7NX/p1716995488309439?thread_ts=1716930104.298129&cid=C06HBA0N7NX